docs/modules/converter/pages/syntax/title.adoc

```diff
@@ -1,16 +1,46 @@
-= Title Slide Customization
+= Title Slide
 :navtitle: Title Slide
```

Comment:
We can remove the `navtitle` attribute since the title of the page is now the same